content/discussions/quickstart.md

```diff
@@ -43,7 +43,7 @@ For more information about providing guidelines for your project, see "[Adding a
 
 ## Creating a new discussion
 
-Anyone with access to a repository can create a discussion.
+Anyone who can view the repository can create a discussion, thus, for public repositories any logged in user start a discussion.
```

Comment:
How about:
```suggestion
Any logged-in user with access to a repository can create a discussion.
```

Reply:
@rcowsill, how about this:
`Any logged-in user who can view the repository can create a discussion.`

Reply:
I made that change - MR should be updated.